packages/graphql-auth-transformer/src/ModelAuthTransformer.ts

```diff
@@ -537,15 +673,19 @@ All @auth directives used on field definitions are performed when the field is r
         const throwIfUnauthorizedExpression = this.resources.throwIfUnauthorized()
 
         // Update the existing resolver with the authorization checks.
-        const templateExpression = compoundExpression([
+        const expressions = [
             staticGroupAuthorizationExpression,
             newline(),
             dynamicGroupAuthorizationExpression,
             newline(),
             ownerAuthorizationExpression,
             newline(),
             throwIfUnauthorizedExpression
-        ])
+        ]
+        expressions.push(
```

Comment:
Do you need this?

Reply:
That was from a prev change removed in latest commit